setup.cfg

```diff
@@ -14,15 +14,17 @@ long_description_content_type = text/markdown
 
 [options]
 install_requires =
-    toml
-    pycparser
-    setuptools
-    prompt_toolkit
-    tqdm
-    jfx_bridge
-    ghidra_bridge
-    psutil
-    pyhidra
+   toml
```

Comment:
tf is going on here

Reply:
Ug i merged the rebase wrong in a few places

Reply:
looks like you just need to add a space to everything here to fix it